examples/games/alien_cake_addict.rs

```diff
@@ -155,25 +155,21 @@ fn setup(mut commands: Commands, asset_server: Res<AssetServer>, mut game: ResMu
     game.bonus.handle = asset_server.load("models/AlienCake/cakeBirthday.glb#Scene0");
 
     // scoreboard
-    commands.spawn(
-        TextBundle::from_section(
+    commands
+        .spawn(TextBundle::from_section(
             "Score:",
             TextStyle {
                 font: asset_server.load("fonts/FiraSans-Bold.ttf"),
                 font_size: 40.0,
                 color: Color::rgb(0.5, 0.5, 1.0),
             },
-        )
-        .with_style(Style {
-            position_type: PositionType::Absolute,
-            position: UiRect {
-                top: Val::Px(5.0),
-                left: Val::Px(5.0),
-                ..default()
-            },
+        ))
+        .insert(PositionType::Absolute)
```

Comment:
Overwriting components via commands like this kinda sucks, and makes me sympathetic to Cart's complaints about `from_section`. That should be a seperate PR though, and I think this is the cleanest way to migrate.

Reply:
Examples are crashing due to duplicate components being inserted in duplicate. Pretty sure it started when changing from `.insert(MyComponent)` to `spawn((SomeBundle, MyComponent))` (In this pr). So will revert back to overwriting components instead since we know that works.